src/snipe/cli/cli_qc.py

```
@@ -2,133 +2,28 @@
 import sys
 import time
 import logging
-from typing import Optional, Any, List, Dict, Set
+from typing import Optional, Any, List, Dict, Set, Union
```

Ruff

`typing.Any` imported but unused

Remove unused import

(F401)